python/BioSimSpace/Parameters/_parameters.py

```diff
@@ -42,16 +42,18 @@
 
 from .._Exceptions import IncompatibleError as _IncompatibleError
 from .._Exceptions import MissingSoftwareError as _MissingSoftwareError
+from .._SireWrappers import Atom as _Atom
 from .._SireWrappers import Molecule as _Molecule
 from ..Solvent import waterModels as _waterModels
 from ..Types import Charge as _Charge
+from ..Types import Length as _Length
 from .._Utils import _try_import, _have_imported
 from .. import _Utils
 
 from ._process import Process as _Process
 from . import Protocol as _Protocol
 
-def parameterise(molecule, forcefield, water_model=None, work_dir=None, property_map={}):
+def parameterise(molecule, forcefield, water_model=None, work_dir=None, property_map={}, **kwargs):
```

Comment:
As far as I can see, `water_model` is not used anywhere.

Reply:
Good catch. I think the parameter shouldn't be there and would be handled by the `kwargs` since it is force field specific. I will probably need to rethink this, as it might be better to have the options handled but silently ignored by functions where they are unsupported. We need to correctly handle people calling the functions directly, or via a generic wrapper when used in a node.